corehq/apps/data_interfaces/urls.py

```diff
@@ -25,12 +25,12 @@
         r'^xform_management/status/(?P<mode>{archive}|{restore})/(?P<download_id>{id_regex})/$'.format(
             archive=FormManagementMode.ARCHIVE_MODE,
             restore=FormManagementMode.RESTORE_MODE,
-            id_regex="[0-9a-fA-Z]{25,32}",
+            id_regex="(?:dl-)?[0-9a-fA-Z]{25,32}",
```

Comment:
Thoughts on making a `DOWNLOAD_ID_REGEX` constant and referencing that in URL patterns?  It would make them a little more verbose, but it'd be more obvious which endpoints use the soil interface.  I could go either way.

Reply:
Good question! It certainly would have made this change easier. I did check to make sure the URLs changed were in fact soil URLs, and that didn't turn out to be too difficult. Concerning whether it's worth it or not, I'm on the fence too.

I do intend to remove this prefix from these URLs at some point in the future once all blob metadata has been migrated, and we know none of the `dl-...` IDs are in use anymore. That will be a two-step operation. One PR will change the place that generates new download IDs. Another PR, deployed several days or weeks later, will remove the prefix from the URL patterns. At that point I will revisit this idea.

Reply:
sounds good to me